cms/envs/common.py

```diff
@@ -546,24 +546,25 @@
 PIPELINE_JS_COMPRESSOR = None
 
 STATICFILES_IGNORE_PATTERNS = (
-    "*.py",
-    "*.pyc"
+    '*.py',
```

Comment:
See note above about unrelated changes. We'll want to undo these changes.

While consistency is a goal, making far reaching changes like this:
- makes it very difficult to review the changes
- makes it harder for us to merge from upstream